config/examples/copyoffload-datamovement-profile.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: nnf.cray.hpe.com/v1alpha4
+data:
+  command: ulimit -n 2048 && mpirun --hostfile $HOSTFILE dcp --progress 1 $SRC $DEST
+  createDestDir: true
+  default: false
+  logStdout: true
+  maxSlots: 0
+  mkdirCommand: mpirun -np 1 --hostfile $HOSTFILE -- mkdir -p $PATH
+  pinned: false
+  progressIntervalSeconds: 5
+  slots: 8
+  statCommand: mpirun -np 1 --hostfile $HOSTFILE -- stat --cached never -c '%F' $PATH
+  storeStdout: true
+kind: NnfDataMovementProfile
+metadata:
+  name: copy-offload-nonmpi
```

Comment:
why nonmpi?

Reply:
Let's let this PR sit while I prepare the PR that has the container profile and then we can see what makes sense.

Reply:
I'm moving this profile to nnf-dm, in https://github.com/NearNodeFlash/nnf-dm/pull/255
You'll see it's paired with its container profile.